README.md

```diff
@@ -47,19 +47,15 @@ bundle install
 
 ### Do this to start development
 ```
-npm start
+npm run dev
 ```
 
+This will serve the aplication on port 8080 and should automatically open http://localhost:8080.
+
 ### Do this to perform a clean build for deployment
 ```
 npm run build
 ```
 NOTE: This isn't necessary as github pages will perform this step automatically.
 
-The website will start at http://localhost:4000
 
-### If you have no styles when building locally
-Will can only make the build work with this command:
-```
-bundle exec jekyll serve --config _config-dev.yml
```

Comment:
I assume your changes break this? I just don't want to pull it if something else doesn't work.

Reply:
Hmm, that's a good point. My change doesn't break this, but rather it just makes it unnecessary as far as I can tell. Rather than removing this I can just change the wording to say something to the effect of "run this command if you need any local config"